src/api/goods.ts

```diff
@@ -5,15 +5,26 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.headers.get('Content-Type')?.includes('application/json')) {
+        // eslint-disable-next-line prefer-promise-reject-errors
+        return Promise.reject('Server response is not json');
+      }
+
+      return response.json();
+    })
+    .catch(error => (`Something went wrong: ${error.message}`));
 }
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => [...goods].sort((firstGood, secondGood) => {
+      return firstGood.name.localeCompare(secondGood.name);
+    })
```

Comment:
you can return this value instantly
```suggestion
    .then(goods => [...goods].sort((firstGood, secondGood) => (
      firstGood.name.localeCompare(secondGood.name);
    ))
```